nix/cbmc/0001-Do-not-download-sources-in-cmake.patch

```diff
@@ -1,30 +1,7 @@
+# Copyright (c) The mlkem-native project authors
+# Copyright (c) The mldsa-native project authors
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
Why is this one `Apache-2.0`?

Reply:
This is from mlkem-native where we have forgotten it. Let's change it there first.